freqtrade/enums/exchangename.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class ExchangeName(Enum):
+    """All the exchanges supported by freqtrade that support leverage"""
+
+    BINANCE = "Binance"
+    KRAKEN = "Kraken"
+    FTX = "FTX"
+    OTHER = None
```

Comment:
I'm not so sure about the name of this enum, but I couldn't think of a better name. Feel free to change it, or it could be a temporary name

Reply:
I'm not sure about this enum at all. 
You can get these names easily from the exchange class - having this as enum again will only cause duplications without many benefits i can see at the moment.

Reply:
The only benefit really is the `OTHER` value, which is for all exchanges that don't have leverage support on freqtrade, but it also might not be necessary